src/retro_data_structures/formats/strg.py

```diff
@@ -347,8 +358,23 @@ def get_language_list(self) -> tuple[str, ...]:
         return tuple(self._raw_languages.keys())
 
     def get_strings(self, language: str = "ENGL") -> tuple[str, ...]:
+        """
+        Gets all strings for the given language.
+        :param language: The language to get. Defaults to ENGL (English).
```

Comment:
can you put newlines between the description and the params list? vscode doesn't display params properly without the empty line